### PR TITLE
Added time slider, time layer aware HOC

### PR DESCRIPTION
**Pull request by hwbllmnn**
Adds a time slider that operates on ISO 8601 time strings. Also adds a HOC that updates layers based on the current time setting.

@terrestris/devs please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -119,6 +119,7 @@
     "metalsmith-layouts": "1.8.1",
     "metalsmith-markdown": "0.2.1",
     "minami": "1.2.3",
+    "moment": "2.20.1",
```
Comment (dnlkoch) [inline package.json]: Should be a `dependency`.

File: src/Slider/TimeSlider.jsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+    /**
+     * Whether to allow range selection.
+     * @type {Boolean}
+     */
+    useRange: PropTypes.bool,
+
+    /**
+     * The default value(s).
+     * @type {any}
```
Comment (dnlkoch) [inline src/Slider/TimeSlider.jsx]: Is there any other type than `Array<String>` or `String` supported?

File: src/Slider/TimeSlider.jsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+    /**
+     * Called when the value changes.
+     * @type {Function}
+     */
+    onChange: PropTypes.func,
+
+    /**
+     * The current value(s).
+     * @type {any}
```
Comment (dnlkoch) [inline src/Slider/TimeSlider.jsx]: See above.

File: src/Slider/TimeSlider.jsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+      moment(val).unix();
+  }
+
+  /**
+   * Formats a timestamp for user display.
+   * @param  {Number} unix unix timestamps
+   * @return {String}      the formatted timestamps
+   */
+  formatTimestamp = unix => {
+    return moment(unix * 1000).format('DD.MM. HH:mm');
```
Comment (dnlkoch) [inline src/Slider/TimeSlider.jsx]: Please make format string configurable.

File: package.json (hunk 1 above)
Comment (hwbllmnn) [reply package.json]: Done.

File: src/Slider/TimeSlider.jsx (hunk 2 above)
Comment (hwbllmnn) [reply src/Slider/TimeSlider.jsx]: Nope, docs adapted.

File: src/Slider/TimeSlider.jsx (hunk 3 above)
Comment (hwbllmnn) [reply src/Slider/TimeSlider.jsx]: Nope, docs adapted.

File: src/Slider/TimeSlider.jsx (hunk 4 above)
Comment (hwbllmnn) [reply src/Slider/TimeSlider.jsx]: Done.

**Pull request merged**